examples/complete/variables.tf

```diff
@@ -25,3 +25,9 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "force_delete" {
```

Comment:
don't expose this in the example - hard code it to `true` and leave a code comment saying its set to true for testing purposes